src/yamlfix/model.py

```diff
@@ -0,0 +1,23 @@
+"""Define program entities like configuration value entities."""
+
+from typing import Optional
+
+from maison.schema import ConfigSchema
+
+
+class YamlfixConfig(ConfigSchema):
+    """Configuration entity for yamlfix."""
+
+    allow_duplicate_keys: bool = True
```

Comment:
I'd say we use `allow_duplicate_keys: bool = False` as default, what do you think?

Reply:
@lyz-code I set this to `False` and enabled it for the one test, that tested on merge keys.